src/main/java/uk/gov/hmcts/reform/em/stitching/service/dto/BundleDTO.java

```diff
@@ -20,7 +20,7 @@
     private boolean hasTableOfContents = true;
     private boolean hasCoversheets = true;
     private boolean hasFolderCoversheets = false;
-
+    private String paginationStyle = "OFF";
```

Comment:
Wasn't this going to be an enum or have you just stubbed it out for now? Also if the values could be camel case e.g. off or topLeft then that would be great